core/src/testils/packet.rs

```diff
@@ -20,44 +20,55 @@ use crate::packets::ip::v4::Ipv4;
 use crate::packets::ip::v6::{Ipv6, SegmentRouting};
 use crate::packets::{Ethernet, Packet, Tcp, Udp};
 
-/// `Packet` extension trait.
+/// [`Packet`] extension trait.
 ///
-/// Methods for packet conversion that make testing less verbose. Does not
-/// guarantee that the result of the conversion will be a valid packet,
+/// Helper methods for packet conversion that make testing less verbose. Does
+/// not guarantee that the result of the conversion will be a valid packet,
 /// and will panic if the conversion fails.
+///
+/// [`Packet`]: crate::packets::Packet
 pub trait PacketExt: Packet + Sized {
+    /// From Mbuf into Ethernet
```

Comment:
description on these are slightly off. they will convert from any packet to the target. they should all say something like `Converts the packet to a BLAH packet`.